src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -383,7 +383,7 @@ private static string ResolvePath(string filename)
                 try
                 {
                     path = Path.Combine(Path.GetDirectoryName(path), filename);
-                    if (File.Exists(path))
+                    if (File.Exists(path) && !Directory.Exists(path))
```

Comment:
How can File.Exists be true, and Directory.Exists also be true - ie., why do you need this check (and the similar one you added below)?

Reply:
I think it is not needed. You should be able to verify your test passes without these.

Reply:
you were right. I didn't need them. I verified the tests passed without them

Thanks